jobs/generation/Utilities.groovy

```diff
@@ -160,6 +160,17 @@ class Utilities {
                                 // For outerloop runs, using Linux kernel version 4.6.2
                                 'outer-linux462': 'auto-auto-ubuntu1604-20160510-20160715outer'
                                 ],
+                            'Ubuntu16.10' :
+                                [
+                                // Generic version label
+                                '' : 'auto-ubuntu1610-20160825',
```

Comment:
I'd just remove this line.  My opinion is now that having empty version names isn't a great idea.